client/state/jetpack-settings/modules/test/actions.js

```diff
@@ -140,4 +149,61 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( '#fetchModuleList', () => {
```

Comment:
If you decide to use the 4 action types, you'll have to update the tests accordingly.